src/test/help.js

```diff
@@ -0,0 +1,40 @@
+/*eslint no-invalid-this: "off" */
+import { exec } from 'child_process'
+
+const usage = `
+  Usage: atomiq [options] [command]
```

Comment:
noted, I think I'll just remove help.js, I added it first as a smoke test, but src/test/api.js is working it's not really needed